apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss

```diff
@@ -13,7 +13,9 @@
 	width: 238px;
```

Comment:
Issue: Slightly unrelated, but the width of the card has been hardcoded to a certain value (`344px`), which on the smallest mobile view, causes the sides of all cards to be cut off.

Reply:
I think we could just reduce the width to 300px